### DISCUSSION TITLE
Cannot create a new snippet on mac

### DISCUSSION THREAD

**Discussion by manurana** [massCodeIO/massCode#50] [General]
14" Macbook Pro

Just downloaded and installed 2.2.0. 
Nothing happens if I press the "+" icon on the top (next to search), or do a <kbd>Cmd</kbd> + <kbd>N</kbd>

In the dev tools, all I can see is this error :
<img width="549" alt="image" src="https://user-images.githubusercontent.com/1749522/164604141-0eb9ffb3-66ec-43fd-9359-2f5bf89a9320.png">

Wondering what could be wrong! Is this a permissions issue on the file system?

(I had tried this with 2.1.0 also, same issue)

**Comment by antonreshetov** [comment]
Just in case, I'll add a link to the issue - https://github.com/massCodeIO/massCode/issues/51

**Comment by antonreshetov** [comment]
In [v2.3.0](https://github.com/massCodeIO/massCode/releases/tag/v2.3.0) added ability to add snippets to Inbox if no folder is selected or any system folder is selected.
